platform-core/src/main/java/com/webank/wecube/platform/core/parser/datamodel/generated/DataModelBaseListener.java

```diff
@@ -0,0 +1,171 @@
+// Generated from /Users/howechen/GitHub/wecube-platform/platform-core/src/main/resources/DataModel.g4 by ANTLR 4.7.2
```

Comment:
Is it possible generate the classes at 'generate-sources' phase to a specified folder instead of committing these generated classes? such that we don't need to commit this kind of classes again when the DataModel.g4 changed.